tests/test_generate.py

```diff
@@ -1,14 +1,23 @@
 import unittest
 import time
 import os
+from elifearticle.utils import unicode_value
 from elifepubmed import generate
 from elifepubmed.conf import config, parse_raw_config
-from elifearticle.utils import unicode_value
+
 
 TEST_BASE_PATH = os.path.dirname(os.path.abspath(__file__)) + os.sep
 TEST_DATA_PATH = TEST_BASE_PATH + "test_data" + os.sep
 generate.TMP_DIR = TEST_BASE_PATH + "tmp" + os.sep
 
+
+def read_file_content(file_name):
```

Comment:
Can just be:
```
def read_file_content(file_name):
    with open(file_name, 'rb') as file_p:
        return file_p.read()
```
The `with` context variable will handle the tidy up of the file.

Reply:
Agreed, I am converting these when I see them - thanks!